### DISCUSSION TITLE
New rank - DD

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#15118] [Ideas] [answered]
You can get this rank if you have pressed all notes by 50 or 100. It looks more like a joke, but this rank is much more difficult to achieve than SS

Example - https://youtu.be/epPa-h7l2KU

**Comment by peppy** [comment, marked as answer]
Dislike. Feels like feature creep to the extreme. Might work well as a mod or something though.

**Comment by smoogipoo** [comment]
Isn't this easy to do via offset?

**Reply by Moltanicaa** [reply]
Yes, but it is worth considering the presence of sliders and spinners
